bigframes/ml/llm.py

```diff
@@ -310,6 +310,64 @@ def predict(
 
         return df
 
+    def score(
+        self,
+        X: Union[bpd.DataFrame, bpd.Series],
+        y=None,  # ignored
```

Comment:
After a second thought, we actually want input_text as X and output_text as y.

Reply:
Done